### PR TITLE
Review more docs

### PR DESCRIPTION
**Pull request by issyl0**
- The one with actual changes is `manage-ruby-dependencies`, ab7cc107.
- The rest are date bumps as they looked OK.

### CODE REVIEW

File: source/manual/manage-ruby-dependencies.html.md [hunk 1]
```diff
@@ -5,29 +5,42 @@ description: How we manage our Ruby dependencies using Dependabot, who can merge
...
-## Security
+### Security
 
 There are 2 safeguards to prevent unauthorised code changes. Firstly, Dependabot can only update the repositories that we [explicitly allow on GitHub][access]. This prevents code changes to other repos. Secondly, we've [set up branch protection](/manual/configure-github-repo.html#auto-configuration) for all repos with the `govuk` label. This prevents Dependabot from writing directly to master.
 
+## Dependabot for security vulnerabilities in dependencies
+
+Since GitHub acquired Dependabot, there are two ways that it updates
+dependencies in your project. Security vulnerabilities come through
+the GitHub dependabot channel under the username `dependabot-preview`.
```
Comment (cbaines) [inline source/manual/manage-ruby-dependencies.html.md]: What's the significance of the `dependabot-preview` username, as this is the one that's used for general Dependabot Pull Requests as well right?

File: source/manual/manage-ruby-dependencies.html.md [hunk 2]
```diff
@@ -5,29 +5,42 @@ description: How we manage our Ruby dependencies using Dependabot, who can merge
...
+
+### Enable "automated security fixes" in the GitHub UI for your repo
+
+1. Go to Security settings for your GitHub repo
+1. Click the "Off: Automated Security Fixes" dropdown and change it to "On"
```
Comment (cbaines) [inline source/manual/manage-ruby-dependencies.html.md]: I don't quite get the relevance of these instructions. Is this something that should be done for all repositories, and what's the effect if they're followed?

File: source/manual/manage-ruby-dependencies.html.md (hunk 1 above)
Comment (issyl0) [reply source/manual/manage-ruby-dependencies.html.md]: Dependabot raises PRs under two different names (one's the "main" one and one's the "GitHub" one). Maybe I got the wrong one for these instructions. Not particularly significant though. I'll remove it.

File: source/manual/manage-ruby-dependencies.html.md (hunk 2 above)
Comment (issyl0) [reply source/manual/manage-ruby-dependencies.html.md]: Maybe we should just link here: https://help.github.com/en/github/managing-security-vulnerabilities/configuring-automated-security-fixes.

Comment (issyl0) [reply source/manual/manage-ruby-dependencies.html.md]: On reflection, I backed out these changes. It's now simple date bump. Maybe in six months GitHub would have integrated Dependabot more and there won't be two separate workflows?

File: source/manual/manage-ruby-dependencies.html.md (hunk 1 above)
Comment (cbaines) [reply source/manual/manage-ruby-dependencies.html.md]: Ok :+1:

File: source/manual/manage-ruby-dependencies.html.md (hunk 2 above)
Comment (cbaines) [reply source/manual/manage-ruby-dependencies.html.md]: Ok :+1:

**Pull request merged**